packages/oui-button/src/index.spec.js

```diff
@@ -0,0 +1,109 @@
+describe('ouiButton', () => {
+  let $componentController
+
+  beforeEach(angular.mock.module('oui.button'))
+
+  beforeEach(inject((_$componentController_) => {
+    $componentController = _$componentController_
+  }))
+
+  describe('Controller', () => {
+    it('should exist', () => {
+      const ctrl = $componentController('ouiButton', {
+        $attrs: {},
+        $element: {},
+        $log: {}
+      })
+
+      expect(ctrl).toBeDefined()
+    })
+  })
+
+  describe('Component', () => {
+    let $rootScope
+    let $compile
+
+    beforeEach(inject((_$rootScope_, _$compile_) => {
+      $rootScope = _$rootScope_
+      $compile = _$compile_
+    }))
+
+    it('should display a button with value of attribute text, and is type="button" by default', () => {
+      let element = angular.element('<oui-button text="foo"></oui-button>')
```

Comment:
This code is duplicated in all tests, we should extract a function e.g.: `getElement('<oui-button text="foo"></oui-button>')`
let element = angular.element('..')
let scope = $rootScope.$new()
$compile(element)(scope)
 scope.$digest()

Reply:
I'll use @neolitec 's helper and update it for my need